crates/tabby-download/src/lib.rs

```diff
@@ -65,7 +65,10 @@ async fn download_file(url: &str, path: &Path) -> Result<()> {
 
     let mut bar = WrappedBar::new(0, url, false);
 
-    https::HTTPSHandler::get(url, &intermediate_filename, &mut bar, "").await?;
+    if let Err(e) = https::HTTPSHandler::get(url, &intermediate_filename, &mut bar, "").await {
+        eprintln!("Fetching model failed: {e}");
```

Comment:
use `tracing::error`, generate error message based on `url` and status code

Reply:
`DownloadError` already gets converted into an error message containing the URL and status code

Reply:
Let's generate the message here, making it easier to create more fine graind err msg in future